torch/_inductor/config.py

```diff
@@ -63,6 +63,10 @@
 # If not specified, a temp directory will be created under the default caching path
 aot_inductor_output_path = ""
 
+# If true, specifies that we are running AOTInductor from an exported program,
+# implying that aot_autograd has already been run.
+aot_from_export = False
```

Comment:
I would prefer to make AOTInductor takes exported program as a mandatory requirement, with some checking in comple_fx_aot. wrapper_call in AotCodeCache is too hacky and not reliable anyway, and I think giving user too much flexibility as in the old way isn't necessarily a good choice.